addons/hr_timesheet/views/project_portal_templates.xml

```diff
@@ -69,7 +69,7 @@
     </template>
 
     <template id="portal_timesheet_table" name="Portal Timesheet Table">
-        <table class="table table-sm">
+        <table class="o_portal_my_doc_table table table-sm">
```

Comment:
Reminder: Use `<t t-if="timesheets" t-call="portal.portal_table">` like is used above in forward port to master, So it'll apply all the styling we want from portal :slightly_smiling_face: